README.md

```diff
@@ -8,6 +8,8 @@
 
 The official [MongoDB](https://www.mongodb.com/) driver for Node.js. Provides a high-level API on top of [mongodb-core](https://www.npmjs.com/package/mongodb-core) that is meant for end users.
 
+**NOTE: v3.0.0-rc0 was recently released with breaking API changes. You can find a list of changes [here](CHANGES_3.0.0.md).**
```

Comment:
maybe let's call it `v3.x was recently released`... so we don't have to update this text constantly